conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -107,3 +119,9 @@ def display_health_checks(prefix: str, verbose: bool = False) -> None:
                 print(f"{package_name}: {len(altered_files)}")
     else:
         print(f"{OK_MARK} There are no packages with altered files.\n")
+
+    print("3. Environment listed in environments.txt file:\n")
+    if check_envs_txt_file(prefix):
+        print("Yes\n")
+    else:
+        print("No\n")
```

Comment:
Let's keep this as concise as possible:

```suggestion
    present = OK_MARK if check_envs_txt_file(prefix) else EX_MARK
    print("3. Environment listed in environments.txt file: {present}\n")
```